### PR TITLE
NoTLS option #minor

### PR DESCRIPTION
**Pull request by jbygdell**
### Describe the pull request:

- [ ] Bug fix
- [ ] Functional change
- [X] New feature
- [ ] Code cleanup
- [ ] Build system change
- [ ] Documentation change
- [ ] Language translation

### Changes made:

1) Updated the docker image to use 3.8.16
2) updated the config semantics to match 3.8.x
3) Entrypoint script no longer creates self signed certificates
4) Adds the NOTLS option to start with SSL disabled

### CODE REVIEW

File: entrypoint.sh [hunk 1]
```diff
@@ -3,562 +3,579 @@
...
+					{source,  [
+											{protocol, amqp091},
+											{uris, ["amqp:///${MQ_VHOST:-}"]},
+											{declarations, [{'queue.declare', [{exclusive, true}] }, {'queue.bind', [{exchange, <<"sda">>}, {queue, <<>>}, {routing_key, <<"inbox">>}] } ] },
+											{queue, <<>>},
+											{prefetch_count, 10}
+										]},
+					{destination, [
+													{protocol, amqp091},
+													{uris, ["amqp:///${MQ_VHOST:-}"]},
```
Comment (blankdots) [inline entrypoint.sh]: is there an extra `/` after all `amqp` ?

File: entrypoint.sh [hunk 2]
```diff
@@ -3,562 +3,579 @@
...
-mkdir -p "/var/lib/rabbitmq/ssl"
-# Generating the SSL certificate + key
-openssl req -x509 -newkey rsa:2048 \
-    -keyout "/var/lib/rabbitmq/ssl/mq-server.key" -nodes \
-    -out "/var/lib/rabbitmq/ssl/mq-server.pem" -sha256 \
-    -days 1000 -subj "${SSL_SUBJ}" && \
-    chmod 600 /var/lib/rabbitmq/ssl/mq-server.*
-fi
+if [ -n "${NOTLS+x}" ]; then
+	echo "Disabeling TLS"
```
Comment (blankdots) [inline entrypoint.sh:7]:
```suggestion
	echo "Disabling TLS"
```

**Review by blankdots — APPROVED** [review]

**Review by blankdots — CHANGES_REQUESTED** [review]
just noticed: https://github.com/neicnordic/sda-mq/blob/master/README.md?plain=1#L3 might require a change as well

**Review by blankdots — APPROVED** [review]
LGTM

not sure we want to remove `hooks` folder with this or another PR

**Review by dbampalikis — APPROVED** [review]
Looks good, although I didn't test it

**Review by norling — APPROVED** [review]
Didn't figure out how test it, but as far as I can tell it looks good!

**Pull request merged**